pages/index.md

```diff
@@ -1054,6 +1054,7 @@
     + [Data Analytics](products/public-cloud-data-analytics)
         + [General information](products/public-cloud-data-analytics-general-information)
             + [Data Analytics Concepts - Security overview](public_cloud/public_cloud_databases/information_01_security_overview)
+            + [Data Analytics Databases - Capabilities and Limitations](public_cloud/public-cloud-data-analytics/information_05_capabilities)
```

Comment:
Rename to "Analytics - Capabilities and Limitations". We will try to follow this template for all the guides "Analytics - XXX" to mimic what was done with "Public Cloud Databases - XXX".